PeerioTechnologies/peerio-desktop
# Release overrides

There's a new directory `release-overrides` (I'm not attached to the name, tell me if I need to change it), which contains overrides to be applied to the file tree before making a production release (that is, 'peerio-release-builder` will apply them before building or before applying overrides from whitelabel repos).

Files inside this directory will ovewrite files inside the root project directory recursively. For example,`release-overrides/src/config-base.js` will overwrite `src/config-base.js`.

NOTE: currently `release-overrides/src/config-base.js` is the same as `src/config-base.js`, the dev/release separation will be applied to it later as needed.

### PR Checklist

- [x] I have merged `dev` into my branch before submitting
- [ ] I have checked with either the repo owner or person who previously worked on this feature to ensure correctness
- [x] I have ensured that I am not using any English strings, only `t('translation_keys)`
- [x] The pull request name follows the standard commit guidelines, e.g. `feat(files): filter by file type`

### Relevant user stories or bugs

### Test plan

* Change something in `release-overrides/src/config-base.js` (e.g. `socketServerUrl`).
* Make a release build.
* Verify that changes in config are applied.

------

The repo owner is @anri-asaturov. The responsible QA specialist is @ebilg.

⬇️ **Note to reviewer:** please squash when merging.
